src/components/00-materials/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 8.0.0
+
+- BREAKING CHANGE: `phone` is now `phone_outlined`; `info-flat` and `plus` were deleted
+- Updated: `check-circle`, `clear`, `date-input`, `add`, `phone` and `info`
```

Comment:
Aren't those breaking changes too? If the size or anything that could break the UI for the user changes, I would highly suggest to als name those as breaking change.

Reply:
Size are all the same. This is not a breaking change in my opinion.